src/commands/devices/change-network.ts

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+import kleur from "kleur";
+
+import { Account } from "@tago-io/sdk";
+
+import { getEnvironmentConfig } from "../../lib/config-file";
+import { errorHandler, infoMSG, successMSG } from "../../lib/messages";
+import { pickDeviceIDFromTagoIO } from "../../prompt/pick-device-id-from-tagoio";
+import { promptTextToEnter } from "../../prompt/text-prompt";
+
+interface BucketSettings {
+  network: string;
+  connector: string;
+}
+
+type environmentConfigResponse = NonNullable<ReturnType<typeof getEnvironmentConfig>>;
+
+
+async function updateDevice(config: environmentConfigResponse, deviceID: string, settings: BucketSettings) {
+  const account = new Account({ token: config.profileToken, region: config.profileRegion });
+
+  const tokens = await account.devices.tokenList(deviceID);
+  const tokenList = tokens.map((token) => token.token);
+
+  if (tokenList) {
+    for (const token of tokenList) {
+      await account.devices.tokenDelete(token);
+    }
+  }
+
+  await account.devices.edit(deviceID, { network: settings.network, connector: settings.connector, active: true });
+
+  for (const token of tokens) {
+    const serieNumber =  token.serie_number as string | undefined;
+    await account.devices.tokenCreate(deviceID, { serie_number: serieNumber, name: token.name, permission: "full" });
+  }
+
+  successMSG(`> ${deviceID} - ${settings.network} network and ${settings.connector} connector updated successfully`);
+}
+
+async function changeNetworkOrConnector(id: string, options: { environment: string; networkID: string; connectorID: string }) {
+  const config = getEnvironmentConfig(options.environment);
+  if (!config || !config.profileToken) {
+    errorHandler("Environment not found");
+    return;
+  }
+
+  let { networkID, connectorID } = options;
+
+  const account = new Account({ token: config.profileToken, region: "usa-1" });
```

Comment:
is this region fixed to "usa-1"? Should it not be using the return type from getEnvironmentConfig (config.profileRegion)?